src/arrayMethodReduce.js

```diff
@@ -4,8 +4,19 @@
  * Implement method Reduce
  */
 function applyCustomReduce() {
-  [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+  [].__proto__.reduce2 = function(callback, ...initialValue) {
```

Comment:
Removed the use of the spread operator (...initialValue) in the function parameter list, as it was unnecessary. The initialValue should be a single value, not an array.

Reply:
👌 fixed